hknweb/templates/base.html

```diff
@@ -125,7 +125,11 @@
     </div>
     {% if user.is_authenticated %}
     <div id="account-nav-bar" class="sub-nav-bar navi-container">
-      <a href="/cand">Profile</a>
+      {% if viewer_is_an_officer %}
```

Comment:
This all seems fine on the front-end! The only thing I'm questioning is how we're determining who is an officer.

Reply:
It checks whether they're in the officer group